src/api/roles/client/client.py

```
@@ -73,20 +75,107 @@ def update_client_information(payload: UpdateClientInfoInput, db = Depends(get_s
 
 @router.post("/me", response_model=ClientAccountResponse)
 def me(db = Depends(get_session), acc: Account = Depends(get_client_account)):
     return ClientAccountResponse(
         base_account=acc,
         client_account=db.get(Client, acc.client_id)
-    )
\ No newline at end of file
+    )
+
+@router.delete("/delete_client_profile", response_model=DunderResponse)
+def delete_client_profile(db = Depends(get_session), acc: Account = Depends(get_client_account)):
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)